### PR TITLE
AV-139825 AV-140770 Multiple default certificates for TLS routes

### PR DESCRIPTION
**Pull request by aaha97**
With these changes users can create an alternate certificate, to allow both RSA and EC default certificates.
This PR also fixes multiple avi objects being generated for default certificates

### CODE REVIEW

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by aaha97** [comment]
build ako

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by chauhanshubham** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Review by chauhanshubham — DISMISSED** [review]

File: internal/nodes/avi_model_l7_translator.go [hunk 1]
```diff
@@ -247,10 +244,24 @@ func (o *AviObjectGraph) BuildTlsCertNode(svcLister *objects.SvcLister, tlsNode
 			utils.AviLog.Infof("key: %s, msg: key not found for secret: %s", key, secretObj.Name)
 			return false
 		}
+		altCert, ok := keycertMap[utils.K8S_TLS_SECRET_ALT_CERT]
```
Comment (akshayhavile) [inline internal/nodes/avi_model_l7_translator.go:247]: One naive doubt: are `alt.crt` and `alt.key`  section part of kubernetes secrets? I didn't get doc for these fields.

File: internal/nodes/avi_model_evh_nodes.go [hunk 2]
```diff
@@ -1130,12 +1130,12 @@ func (o *AviObjectGraph) BuildTlsCertNodeForEvh(svcLister *objects.SvcLister, tl
 		secretNS = namespace
 	}
 
-	if isSecretK8sSecretRef(secretName) {
+	if lib.IsSecretK8sSecretRef(secretName) {
 		secretName = strings.Split(secretName, "/")[2]
 	}
-
+	var altCertNode *AviTLSKeyCertNode
 	certNode := &AviTLSKeyCertNode{
```
[the remark this answers is not in the document]
Comment (aaha97) [reply internal/nodes/avi_model_evh_nodes.go]: fixed

**Review by chauhanshubham — DISMISSED** [review]

**Comment by aaha97** [comment]
build ako

**Comment by akshayhavile** [comment]
@aaha97: Failure in UT has been fixed as part of PR #813. After merge, we can run UT again.

**Comment by aaha97** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Comment by aaha97** [comment]
build ako

**Review by chauhanshubham — APPROVED** [review]

**Pull request merged**